examples/stm32/f3/nucleo-f303re/fancyblink/README.md

```diff
@@ -0,0 +1,7 @@
+# README
+
+This is a small example program using libopencm3.
+
+It's intended for the ST Nucleo-F303RE eval board. It should blink
+LEDs connected to the Arduino header, D6~D13.
```

Comment:
fancy blink traditionally is just a more convolved example of miniblink, but still using the leds on board.  If the nucleo doens't have any extra leds, I dont' see any point in an extra led blinker.